src/utils/utils.js

```diff
@@ -1,5 +1,9 @@
-// import * as XMLHttpRequestNode from "Xmlhttprequest";
+/* eslint-disable unicorn/no-for-loop */
+/* eslint-disable sonarjs/no-redundant-boolean */
+/* eslint-disable eqeqeq */
+/* eslint-disable no-param-reassign */
 import get from 'get-value';
+import set from 'set-value';
```

Comment:
I believe this package is impacting the size of the core SDK as well. Can you move it out to somewhere else, perhaps `commonUtils.js`?

Reply:
removed usage of this package and it's not necessary to use